tests/testTriggers.go

```diff
@@ -7211,6 +7213,163 @@ func TriggerAsyncDRMigrationSchedule(contexts *[]*scheduler.Context, recordChan
 	})
 }
 
+func TriggerMetroDRMigrationSchedule(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer endLongevityTest()
+	startLongevityTest(MetroDRMigrationSchedule)
+	defer ginkgo.GinkgoRecover()
+	log.InfoD("Metro DR Migrationschedule triggered at: %v", time.Now())
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: MetroDRMigrationSchedule,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	var (
+		migrationNamespaces      []string
+		taskNamePrefix           = "metro-dr-mig-sched"
+		clusterDomainWaitTimeout = 10 * time.Minute
+		defaultWaitInterval      = 10 * time.Second
+		allMigrationsSched       = make(map[string]string)
+		includeVolumesFlag       = false
+		includeResourcesFlag     = true
+		startApplicationsFlag    = false
+		scpolName                = "async-policy"
+		suspendSched             = false
+		autoSuspend              = false
+		schd_pol                 *storkapi.SchedulePolicy
```

Comment:
schPol since we are using camel case everywhere

Reply:
Done!